### ISSUE TITLE
Security example type checks (but shouldn't)

### ISSUE THREAD

**Issue by GuiltyDolphin** [granule-project/granule#100]
When trying the example from https://granule-project.github.io/security.html.

```
secret : Int [Private]
secret = [42]

hash : forall {l : Level} . Int [l] -> Int [l]     -- at any level...
hash [x] = [x * x * x]                             -- ...hash by cubing

main : Int [Public]
main = hash secret
```

I get:

```
Checking hashLevel.gr...
OK, evaluating...
74088
```

Rather than the expected typing error.

**Comment by dorchard** [comment]
It appears that at some point recently the lattice has got flipped (try switching Private and Public here). Not sure how. Will look into this at the end of the month I hope.

**Comment by dorchard** [comment]
Fixed by 0f58138af41dc9e761e40eedbd0c37395552c2c8
